README.md

```diff
@@ -1,141 +1,114 @@
-* [Ansible Collection Development Guide](https://docs.ansible.com/ansible/devel/dev_guide/developing_collections.html#contributing-to-collections)
+### Plugins
 
-## Collection maintenance
+* `kubevirt`: inventory source for kubevirt virtual machines
+* `kubevirt_vm`: create or delete kubevirt virtual machines
 
-The current maintainers are listed in the [MAINTAINERS](MAINTAINERS) file. If you have questions or need help, feel free to mention them in the proposals.
+## Using this collection
 
-To learn how to maintain / become a maintainer of this collection, refer to the [Maintainer guidelines](MAINTAINING.md).
+<!--start galaxy_download -->
+### Installing the Collection from Ansible Galaxy
 
-## Governance
+Before using this collection, you need to install it with the Ansible Galaxy command-line tool:
+```bash
+ansible-galaxy collection install kubevirt.core
+```
+<!--end galaxy_download -->
 
-<!--Describe how the collection is governed. Here can be the following text:-->
+### Build and install locally
 
-The process of decision making in this collection is based on discussing and finding consensus among participants.
+Clone the repository, checkout the tag you want to build, or pick the main branch for the development version; then:
+```bash
+ansible-galaxy collection build .
+ansible-galaxy collection install kubevirt-kubevirt.core-*.tar.gz
+```
 
-Every voice is important. If you have something on your mind, create an issue or dedicated discussion and let's discuss it!
+### Dependencies
 
-## Tested with Ansible
+#### Ansible collections
 
-<!-- List the versions of Ansible the collection has been tested with. Must match what is in galaxy.yml. -->
+* [kubernetes.core](https://docs.ansible.com/ansible/latest/collections/kubernetes/core/index.html)
 
-## External requirements
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+```yaml
+---
+collections:
+  - name: kubevirt.core
+```
 
-<!-- List any external resources the collection depends on, for example minimum versions of an OS, libraries, or utilities. Do not list other Ansible collections here. -->
-- python >= 3.6
+#### Python libraries
 
-Python libraries:
 - kubernetes
 - PyYaml
 - jsonpatch
 - jinja2
 
-### Supported connections
-<!-- Optional. If your collection supports only specific connection types (such as HTTPAPI, netconf, or others), list them here. -->
+To install all the dependencies:
+```bash
+pip install -r requirements.txt
+```
 
-## Included content
+See [Ansible Using collections](https://docs.ansible.com/ansible/devel/user_guide/collections_using.html) for more details.
 
-<!-- Galaxy will eventually list the module docs within the UI, but until that is ready, you may need to either describe your plugins etc here, or point to an external docsite to cover that information. -->
+<!--start community_readme -->
+## Code of Conduct
```

Comment:
https://github.com/kubevirt/kubevirt.core/blob/main/CODE_OF_CONDUCT.md

This says we follow the KubeVirt COC. Can we keep it as this repo lives under the KubeVirt org?